docs/pages/example/mapbox-gl-rtl-text.js

```diff
@@ -3,7 +3,7 @@ title: Add support for right-to-left scripts
 description: >-
   Use the [mapbox-gl-rtl-text](https://github.com/mapbox/mapbox-gl-rtl-text)
   plugin to support scripts that use right-to-left layout (such as Arabic or
-  Hebrew). Note that Mapbox Studio does not yet support RTL text.
+  Hebrew).
```

Comment:
By the way, what’s left of this paragraph can be misleading. Arabic and Hebrew are the only right-to-left scripts that gl-language reliably supports. Moreover, Arabic is the only Arabic-script language it supports.

/ref https://github.com/mapbox/documentation/issues/156#issuecomment-477664649

Reply:
That's really interesting @1ec5, but I'd prefer to leave this change for a separate content update – sounds like we need to workshop the precise language we want to use when describing RTL and apply it across our docs.